src/shared/components/namespaceColumns/buildNamespaceColumnConfig.ts

```diff
@@ -0,0 +1,31 @@
+import { NamespaceColumnConfig } from 'shared/components/namespaceColumns/NamespaceColumnConfig';
+import _ from 'lodash';
+import { WithNamespace } from 'shared/components/namespaceColumns/NumericNamespaceColumnFormatter';
+
+export function buildNamespaceColumnConfig(
+    mutations: WithNamespace[]
+): NamespaceColumnConfig {
+    if (!mutations) {
+        return {};
```

Comment:
Dit moet toch niet in losse files. Maak liever een namespaceUtils.tx file.

Reply:
Fixed in [112e373](https://github.com/cBioPortal/cbioportal-frontend/pull/4418/commits/112e37320fb14c6d8270d421ec5e7a323a05aea3)